src/enums/Colors.ts

```diff
@@ -1,12 +1,10 @@
 /**
  * Enum of chess piece colors.
- * @readonly
- * @enum {number}
  */
-enum ColorsEnum {
+enum Colors {
     Invalid = 0,
```

Comment:
Jaki jest zamysł za tym Invalid (nie twierdze że jest nie potrzebne tylko nie bardzo rozumiem jak to mamy wykorzystać)

Reply:
Tak mnie uczono, że każdy stan definiowany przez enuma powinien mieć wartość invalid. Ale może to domena systemów wbudowanych